app/javascript/components/App.js

```diff
@@ -20,26 +20,107 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      parties: [],
-      items: [],
+      party_id: '',
+      parties: '',
+      items: '',
     };
   }
 
+  componentDidMount() {
+    this.eventsIndex();
+    this.itemsIndex();
+  }
+
+  eventsIndex = () => {
+    fetch('http://localhost:3000/parties')
+      .then((response) => {
+        return response.json();
+      })
+      .then((partiesArray) => {
+        console.log(partiesArray);
+        this.setState({ parties: partiesArray });
+      })
+      .catch((errors) => {
+        console.log('index errors:', errors);
+      });
+  };
+
+  itemsIndex = () => {
+    fetch('http://localhost:3000/items')
+      .then((response) => {
+        return response.json();
+      })
+      .then((itemsArray) => {
+        console.log(itemsArray);
+        this.setState({ items: itemsArray });
+      })
+      .catch((errors) => {
+        console.log('index errors:', errors);
```

Comment:
```suggestion
        console.error('index errors:', errors);
```
could help with signaling